### PR TITLE
Maks dev

### PR DESCRIPTION
**Pull request by msioss**
Seeder#1 pull

### CODE REVIEW

File: EJournal/Data/SeedData/PreConfigured.cs [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+                        await context.BaseProfiles.AddAsync(prof);
+                        await context.SaveChangesAsync();
+                        await context.StudentProfiles.AddAsync(new StudentProfile { Id = prof.Id });
+                        await context.SaveChangesAsync();
+                    }
+                }
+            }
+            catch (Exception ex)
+            {
+
```
Comment (vunt1k) [inline EJournal/Data/SeedData/PreConfigured.cs:341]: where is the code 2

File: EJournal/Data/SeedData/PreConfigured.cs [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+                    BaseProfile profile2 = new BaseProfile
+                    {
+                        Id = user2.Id,
+                        Name = "Лариса",
+                        LastName = "Осадча",
+                        Surname = "Костянтинівна",
+                        Adress = "Pushkina 44, 12a",
+                        DateOfBirth = new DateTime(1983, 6, 23),
+                    };
+
```
Comment (vunt1k) [inline EJournal/Data/SeedData/PreConfigured.cs]: 3

File: EJournal/Data/SeedData/PreConfigured.cs [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+                        PhoneNumber = "+380503334031",
+                    };
+
+                    DbUser user2 = new DbUser
+                    {
+                        UserName = "emdeputy",
+                        Email = "emdeputyninja@gmail.com",
+                        PhoneNumber = "+380505551541",
+                    };
+
```
Comment (vunt1k) [inline EJournal/Data/SeedData/PreConfigured.cs]: 2

File: EJournal/Data/SeedData/PreConfigured.cs [hunk 4]
```diff
@@ -0,0 +1,352 @@
...
+            {
+                if (!context.TeacherProfiles.Any())
+                {
+                    DbUser user1 = new DbUser
+                    {
+                        UserName = "director",
+                        Email = "beedirector@gmail.com",
+                        PhoneNumber = "+380503334031",
+                    };
+
```
Comment (vunt1k) [inline EJournal/Data/SeedData/PreConfigured.cs]: 1

File: EJournal/Data/SeedData/PreConfigured.cs [hunk 5]
```diff
@@ -0,0 +1,352 @@
...
+            }
+            catch (Exception)
+            {
+
+            }
+
+        }
+        public static async Task SeedUsers(UserManager<DbUser> userManager, EfDbContext context)
+        {
+
```
Comment (vunt1k) [inline EJournal/Data/SeedData/PreConfigured.cs]: empty space

File: EJournal/Data/SeedData/PreConfigured.cs [hunk 6]
```diff
@@ -0,0 +1,352 @@
...
+                    roleName = "CycleCommisionHead";
+                    var result9 = roleManager.CreateAsync(new DbRole
+                    {
+                        Name = roleName
+                    }).Result;
+                }
+            }
+            catch (Exception)
+            {
+
```
Comment (vunt1k) [inline EJournal/Data/SeedData/PreConfigured.cs:71]: where is the code?

File: EJournal/Data/SeedData/PreConfigured.cs [hunk 7]
```diff
@@ -0,0 +1,352 @@
...
+using EJournal.Data.Entities;
+using EJournal.Data.Models;
+
+namespace EJournal.Data.SeedData
+{
+    public class PreConfigured
+    {
+        public static void SeedRoles(RoleManager<DbRole> roleManager)
+        {
+
```
Comment (vunt1k) [inline EJournal/Data/SeedData/PreConfigured.cs]: Delete empty space

**Review by vunt1k — APPROVED** [review]

**Review by Maksym-Rohach — APPROVED** [review]

**Pull request merged**